src/clj/ataru/forms/form_access_control.clj

```diff
@@ -32,11 +32,23 @@
   (let [form-keys-from-tarjonta (set (keys (tarjonta-protocol/get-forms-in-use virkailija-tarjonta-service session)))]
     (contains? form-keys-from-tarjonta form-key)))
 
-(defn get-organizations-with-edit-rights [session]
-  (-> session
-      :identity
-      :user-right-organizations
-      :form-edit))
+(defn get-organizations-with-edit-rights
+  [session]
+  (if (:selected-organization session)
+    (if (-> session :identity :superuser)
```

Comment:
```
(if (or (-> session :identity :superuser)
        (contains? (-> session :selected-organization :rights) :form-edit))
  [(:selected-organization session)]
  [])
```
pitäis käsittääkseni riittää.

Reply:
Joo. Luulen, että tässä oli alun perin ideana tarkistaa oikeuksia koko hierarkiasta, mutta sitähän ei tässä kohdin tarvitse tehdä ja homma jäi vaan vaiheeseen.